drivers/input/touchscreen/edt-ft5x06.c

```diff
@@ -1292,6 +1324,7 @@ static int edt_ft5x06_ts_probe(struct i2c_client *client,
 	}
 
 	if (tsdata->reset_gpio) {
+		gpiod_set_value_cansleep(tsdata->reset_gpio, 1);
```

Comment:
Not necessary as you've already had
```
 	tsdata->reset_gpio = devm_gpiod_get_optional(&client->dev,
						     "reset", GPIOD_OUT_HIGH);
```
above

Reply:
Ah. Thanks. Will remove it.